test/IISIntegration.FunctionalTests/IISIntegration.FunctionalTests.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
I was just reverting the previous migration change. We used to cross-compile this due to https://github.com/aspnet/IISIntegration/pull/345/files#r108019658 but that may no longer be relevant. Will test once the BasicMiddleware packages are available on the feed.